app/services/data/update-assignment-of-claims.js

```diff
@@ -0,0 +1,10 @@
+const config = require('../../../knexfile').intweb
+const knex = require('knex')(config)
+const dateFormatter = require('../date-formatter')
+const environmentVariables = require('../../../config')
+
+module.exports = function (claimId, assignedTo) {
+  return knex('Claim')
+    .where({'ClaimId': claimId})
+    .update({ 'AssignedTo': assignedTo, 'AssignmentExpiry': dateFormatter.now().add(environmentVariables.ASSIGNMENT_EXPIRY_TIME, 'minutes').toDate() })
```

Comment:
update LastUpdated so it is checked at the point of assignment to prevent two people assigning themselves and not realising